backend/experiment/actions/consent.py

```diff
@@ -64,7 +66,7 @@ class Consent(BaseAction):  # pylint: disable=too-few-public-methods
                 amet, nec te atqui scribentur. Diam molestie posidonium te sit, \
                 ea sea expetenda suscipiantur contentiones."
 
-    def __init__(self, text: File, title="Informed consent", confirm="I agree", deny="Stop", url=""):
+    def __init__(self, text: File, title: str="Informed consent", confirm: str="I agree", deny: str="Stop", url: str=""):
```

Comment:
Might also explicitly state the return type:
```suggestion
    def __init__(self, text: File, title: str="Informed consent", confirm: str="I agree", deny: str="Stop", url: str="") -> dict:
```